.circleci/config.yml

```diff
@@ -1,21 +0,0 @@
-version: 2.1
```

Comment:
This merge commit doesn't seem to be a merge

Reply:
Sorry that was a bad merge, now all conflicts are resolved and github CI is updated to use `cargo test --all-features`, plz consider square merging this PR since it has many unuseful merge commits, thx!

Reply:
No worries :slightly_smiling_face:

We've found that squash merging caused us real headache (at least, in the mina repo) when resolving conflicts between longer-lived branches. Our current flow is to `git merge` for PR merges, and to bring branches up-to-date. This also allows us to build 'PR chains' of branches of dependent changes.

History ends up a bit messier, but it's a small price to pay compared to the amount of time we used to spend handling merge conflicts and branch drift!